README.md

```diff
@@ -72,6 +72,10 @@ The tests support a few environment variables:
 - `ATHENA_DATABASE` can be used to override the default database "go_athena_tests"
 - `S3_BUCKET` can be used to override the default S3 bucket of "go-athena-tests"
 - `ATHENA_REGION` or `AWS_DEFAULT_REGION` can be used to override the default region of "us-east-1"
+- `ATHENA_WORK_GROUP` can be used to override the default workgroup of "primary"
+- `ATHENA_AUTO_OUTPUT_LOCATION` is a parameter for test of getting output_location value from workgroup
```

Comment:
> `ATHENA_AUTO_OUTPUT_LOCATION` is a parameter for test

I might be wrong, but it seems like this parameter exists for tests from the explanation. Is it correct?

I guess the parameter is actually useful if we wanna get `output_location` from a specified `workgroup` regardless of whether it is a test.

Reply:
Thank you for your feedback!

> I might be wrong, but it seems like this parameter exists for tests from the explanation. Is it correct?

Exactly!

> I guess the parameter is actually useful if we wanna get output_location from a specified workgroup regardless of whether it is a test.

Can you tell me why you think it's useful?
Is your suggestion that this environment variables should be specified outside of test?

Reply:
Sorry, I forgot these environment variables are for testing!
But as we discussed, it might be better to rename these variables to clarify these are for testing :)

Reply:
@danimal141 
I deleted the parameter of `ATHENA_AUTO_OUTPUT_LOCATION` and changed test.

- The test should cover all cases even if ATHENA_AUTO_OUTPUT_LOCATION does not exist
- To maintain backward compatibility with applications that were written with the older [go-athena](https://github.com/segmentio/go-athena)